common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -782,12 +794,14 @@ class @Problem
         .add(@saveButton)
         .add(@hintButton)
         .add(@showButton)
+        .removeClass('is-disabled')
```

Comment:
Yes, nice catch.  Thank you.  I know we went back and forth on this.  I will match the submit branch on this line.